Library/Tracking/KSRAnalytics.swift

```diff
@@ -1810,9 +1810,13 @@ private func userProperties(for user: User?, config _: Config?, _ prefix: String
   var props: [String: Any] = [:]
 
   props["backed_projects_count"] = user?.stats.backedProjectsCount
-  props["created_projects_count"] = user?.stats.createdProjectsCount
+  props["created_projects_count"] = (user?.stats.createdProjectsCount ?? 0) +
+    (user?.stats
+      .draftProjectsCount ??
+      0) // Stats.createdProjectsCount is the count of projects user has lauched only, while event property `created_projects_count` includes Stats.createdProject + Stats.draftProjectsCount
```

Comment:
typos and still unclear. move this above line 1813 with the following description:

the product/insights team definition of `created_projects_count` is the sum of `createdProjectsCount` and `draftProjectsCount`